docs_source_files/content/webdriver/web_element.de.md

```diff
@@ -534,6 +535,7 @@ Wird verwendet um das Element, das den Fokus hat zu finden oder um es zu verfolg
   import org.openqa.selenium.By
   import org.openqa.selenium.chrome.ChromeDriver
 
+
```

Comment:
Hi @AgileTestingAlliance,

I think we don't need an extra black line. Could you please remove the extra blank lines added in the code?